projects/plugins/jetpack/class.jetpack.php

```diff
@@ -6728,9 +6728,14 @@ public static function get_products_for_purchase( $show_legacy = false ) {
 			'features'          => array(
 				_x( 'Real-time cloud backups', 'Backup Product Feature', 'jetpack' ),
 				_x( '10GB of backup storage', 'Backup Product Feature', 'jetpack' ),
-				_x( '30-day archive & activity log', 'Backup Product Feature', 'jetpack' ),
+				_x( '30-day archive & activity log*', 'Backup Product Feature', 'jetpack' ),
 				_x( 'One-click restores', 'Backup Product Feature', 'jetpack' ),
 			),
+			'disclaimer'        => array(
+				'text'      => __( '* Subject to your usage and storage limit.', 'jetpack' ),
+				'link_text' => __( 'Learn more', 'jetpack' ),
+				'url'       => 'https://cloud.jetpack.com/pricing#backup-storage-limits-faq',
```

Comment:
Rather than hard-coding in the link here, I think we need to use Jetpack Redirects here: PCYsg-pY7-p2
You would first create the link in the Jetpack Redirect manager, and then here in the code, you would reference the link you created by using `Redirect::get_url(...)`

Reply:
Always so much to learn. Hopefully my brain can hold onto these things 🧠